city_scrapers/spiders/il_sports_facilities_authority.py

```diff
@@ -0,0 +1,79 @@
+from datetime import datetime, timedelta
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class IlSportsFacilitiesAuthoritySpider(CityScrapersSpider):
+    name = "il_sports_facilities_authority"
+    agency = "Illinois Sports Facilities Authority"
+    timezone = "America/Chicago"
+    allowed_domains = ["www.isfauthority.com"]
+    start_urls = ["https://www.isfauthority.com/governance/board-meetings/"]
+    location = {
+        'name': 'Authority offices',
+        'address': 'Guaranteed Rate Field, 333 West 35th Street, Chicago, IL'
+    }
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+        """
+        self._validate_location(response)
+        for item in response.css(
+            'div.wpb_text_column div.wpb_wrapper div.inner-text h2, '
+            'div.wpb_text_column div.wpb_wrapper p'
+        ):
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description='',
+                classification=BOARD,
+                start=self._parse_start(item),
+                end=None,
+                all_day=False,
+                time_notes='See agenda for meeting time',
+                location=self.location,
+                links=self._parse_links(item),
```

Comment:
From the output of the CI run it looks like all of the links are getting associated with one meeting: https://github.com/City-Bureau/city-scrapers/runs/365507248#step:11:73

Not sure what caused this, but the selector I gave you might have been off